src/components/TripCard/TripCard.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const TripCard = ({image, name, date, sunrise, sunset}) => {
+
+  return(
+    <article className='trip-card'>
+      <img src={image.url} alt={image.altText} />
+      <div className='trip-info'>
+        <h3>{name}</h3>
+        <p>{date}</p>
+        <p>{sunrise}</p>
+        <p>{sunset}</p>
+      </div>
+    </article>
+  )
+
+}
+
+export default TripCard;
```

Comment:
Since you have functional components you could save a line and "export const TripCard", you'd just have to change the import to {TripCard} instead of TripCard :)